doc/perlvars.pm

```diff
@@ -0,0 +1,129 @@
+#! /usr/bin/env perl
+# Copyright 2019 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+
+# Set some Perl variables for use by util/dofile.pl when processing
+# POD files (mainly man1).
+
+# Verify options
+$OpenSSL::safe::opt_v_synopsis = ""
+. "[B<-attime> I<timestamp>]\n"
+. "[B<-check_ss_sig>]\n"
```

Comment:
So that's a lot of quotes and explicit newlines to deal with any time these texts need editing. Another way to create these strings is:

``` perl
$OpenSSL::safe::opt_v_synopsis = <<'_____';
[B<-attime> I<timestamp>]
[B<-check_ss_sig>]
...
[B<-x509_strict>]
[B<-certfile> I<file>]
_____
```

I leave it to you to decide how you want to go about that.

Reply:
I tried to do that, but in order to get the spacing right I need to not having the trailing newline so I figured this was simplest.

Reply:
Okie